cypress/integration/pages/articles/tests.js

```diff
@@ -31,7 +31,40 @@ export const testsThatFollowSmokeTestConfig = ({
   pageType,
   variant,
 }) => {
+  let articlesData;
   describe(`Running tests for ${service} ${pageType}`, () => {
+    before(() => {
+      const env = Cypress.env('APP_ENV');
+      if (env !== 'local') {
+        const scriptSwitchServices = ['serbian', 'ukchina', 'zhongwen'];
+        let appendVariant = '';
+
+        // eslint-disable-next-line prefer-destructuring
+        const articleId = Cypress.env('currentPath')
+          .split('articles/')
+          .pop()
+          .split('?')[0];
```

Comment:
May be worth using something like:
`Cypress.env('currentPath').match(/(c[a-zA-Z0-9]{10}o)/)?.[1];`

This is the same regex used in the BFF fetcher to extract the ID as it works with `/variant` at the end too.

Reply:
I have replaced with this regex. I might learn more about this on a learning day! it looks cool